src/blending-machine/index.js

```diff
@@ -82,7 +82,7 @@ var blendingMachine = {
         };
 
         fluidAPI.on.and = fluidAPI;
```

Comment:
Can you please add new lines to support chaining whens and thens on the given (backward compatibility)?

Like this:
fluidAPI.given.then = fluidAPI.then
...